### PR TITLE
D2 helmets extraction in Go

### PR DESCRIPTION
**Pull request by AndriiMoskvin**
- [x] Manifest Ru Database (both JSON and SQL)
- [x] One item example extracted from database
- [x] `Forsarion` inventory added
- [x] Endpoint to load manifest.json
- [x] Endpoint to extract helmets from `Forsarion` inventory

screen shot 2019-01-02 at 12 25 19 pm

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,79 +1,12 @@
...
-
-# Visual Studio Code
-
-.vscode/*
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
-!.vscode/extensions.json
\ No newline at end of file
+# Virtualenv
```
Comment (oleksii-demedetskyi) [inline .gitignore:1]: Do we need this on top level? Or it is a mistake?

File: d2-roll.code-workspace [hunk 2]
```diff
@@ -7,6 +7,10 @@
 		{
 			"name": "Web",
 			"path": "./web"
+		},
+		{
```
Comment (oleksii-demedetskyi) [inline d2-roll.code-workspace]: Why do we need Root?

**Review by oleksii-demedetskyi — COMMENTED** [review]
Can you provide a little bit about direction and goals?

**Comment by oleksii-demedetskyi** [comment]
Also, Bungie have json manifest. Can we use it and avoid mapping etc altogether?

File: d2-roll.code-workspace (hunk 2 above)
Comment (Forsarion) [reply d2-roll.code-workspace]: No - need dropped it.

File: .gitignore (hunk 1 above)
Comment (Forsarion) [reply .gitignore:1]: Docker don't works on my machine without virtual env - I've added this to avoid committing bunch of machine-specific stuff.

**Comment by Forsarion** [comment]
I will work with a JSON as you suggested - will try to write some Go code to extract Helmets.

**Comment by Forsarion** [comment]
@AlexeyDemedetskiy I've created endpoint that finds helmets. I think we can merge this PR and continue moving with plugs.

**Comment by oleksii-demedetskyi** [comment]
🚀

**Pull request merged**